### PR TITLE
Moves the bag of holding up one tech tier

### PR DESCRIPTION
**Pull request by ghost**
:cl: Floyd / Qustinnus
balance: the bag of holding now requires a higher level of research to be unlocked as well as janitor tech (trashbags of holding)
balance: the bluespace bodybag is now unlocked with mini BS research
/:cl:

bounty for @Yakumochem

wanted to do it earlier but fell asleep so here it is

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
needs a justification

**Comment by Qustinnus** [comment]
From yakumo "As for the reasoning behind the bag changes, I've never liked the fact you unlock then wither tier 4 parts. The parts are always useful, but the bags tend to be more trouble then they're worth, and a pretty powerful item in their own right."

**Comment by Whoneedspacee** [comment]
this research tech is busted for its value anyways but im not sure it should be up a tech tier and not just a seperate branch entirely, maybe bluespace storage for all those types of things

**Comment by wesoda25** [comment]
this is just an issue with BOH bombing i think, which a nerf is up rn, i think tech tier is fine

**Comment by dr-bee** [comment]
This brings BOH back to what is was pre-techwebs. Before it required a little more bluespace tech than was needed for tier 4 parts, so you could purposefully choose not to research them.

**Comment by Tlaltecuhtli** [comment]
what does waiting 2 more minutes bring to the game

**Comment by Trilbyspaceclone** [comment]
image
Maybe something like this?

**Comment by Qustinnus** [comment]
I'll wait on yakumo to say what they want

**Comment by YakumoChen** [comment]
>This brings BOH back to what is was pre-techwebs. Before it required a little more bluespace tech than was needed for tier 4 parts, so you could purposefully choose not to research them.

This is entirely the reason. It is also a part of a series of code bounties I have issued for which the ultimate goal is for the research director to be able to have more control over research including preventing research they don't like.

**Comment by YakumoChen** [comment]
> 
> 
> what does waiting 2 more minutes bring to the game

points are easy to spend and hard to save up for. You can choose not to buy the bag tech and spend the points on other things. There are a lot of techs and it is easy to choose to de-prioritize stuff you don't want, or at the very least delay it getting out.

**Comment by Tlaltecuhtli** [comment]
wasnt pre techweb RD able to disable designs from being made?

**Comment by YakumoChen** [comment]
> 
> 
> wasnt pre techweb RD able to disable designs from being made?

Yes, that's the goal overall.

**Comment by ExcessiveUseOfCobblestone** [comment]
If the bluespace storage is moving to it's own tree then the SoHs can be put in there or the tech be a prereq. Regardless it should be associated with the other items imo.

May want to follow up with Yakumo and see if they are ok with the current iteration or if they're wanting you to make a new tree for it. I don't care which as long as it's not made easier to obtain.

**Comment by YakumoChen** [comment]
@ExcessiveUseOfCobblestone I don't see the need to make trash bags or miner bags nigh-impossible to unlock, but I could see it with body bags.

@Qustinnus please add the janitor node to prereq and move bluespacebodybag to micro_bluespace

Otherwise I'm perfectly happy with how the current balance is and don't see a need to over-complicate the idea of "make bags of holding harder to get", which is all part of a greater scheme anyway.

Bags of Holding are the biggest problem I have and the nerf fell through anyway, this is a more subtle nerf that will have better meaning later.

**Comment by Tlaltecuhtli** [comment]
why janitor tech pre req?

**Comment by YakumoChen** [comment]
@Tlaltecuhtli because of trash bags of holding. It makes sense when you think about it.

**Comment by ExcessiveUseOfCobblestone** [comment]
> @ExcessiveUseOfCobblestone I don't see the need to make trash bags or miner bags nigh-impossible to unlock, but I could see it with body bags. I also don't see a need to make a whole tree when bluespace tech makes sense as it is.

I just think it's more of an intuitive "you research the smaller versions first to unlock the big mama" reasoning rather than it being based on the item being op/up

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -199,16 +199,25 @@
 	display_name = "Miniaturized Bluespace Research"
 	description = "Extreme reduction in space required for bluespace engines, leading to portable bluespace technology."
 	prereq_ids = list("bluespace_travel", "practical_bluespace", "high_efficiency")
-	design_ids = list("bluespace_matter_bin", "femto_mani", "triphasic_scanning", "bag_holding", "quantum_keycard", "wormholeprojector")
+	design_ids = list("bluespace_matter_bin", "femto_mani", "bluespacebodybag", "triphasic_scanning", "quantum_keycard", "wormholeprojector")
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/research/techweb/all_nodes.dm]: undoc'd

**Pull request merged**